hoomd/__init__.py

```diff
@@ -104,4 +104,7 @@ def _hoomd_sys_excepthook(type, value, traceback):
 
 # Install a SIGTERM handler that gracefully exits, allowing open files to flush
 # buffered writes and close.
```

Comment:
Probably worth mentioning why we are using the try block here as it is non-obvious.